### PR TITLE
Bug Fix: Unable to logout after initial login ( closes 1581 ) 

### LINKED ISSUE #1581: Unable to logout after initial login
**Describe the bug**
After an initial log in, you are unable to log out from the Talawa application.

**To Reproduce**

[Follow the steps outlined here till the 50 second mark](https://www.youtube.com/watch?v=ix3eSReJj2k?t=23)

**Expected behavior**
Clicking the logout option should take you to the login screen.

**Actual behavior**
Choosing the logout option takes you to the screen you're currently on.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by SiddheshKukade**
**What kind of change does this PR introduce?**

Logout feature was not working I've fixed the issue.

**Issue Number:**

Fixes #1581 

**Did you add tests for your changes?**

Not required
**Snapshots/Videos:**

https://user-images.githubusercontent.com/65951872/222962278-29ed0342-0384-4e5c-9826-2134195a88c3.mp4

 
 
**Does this PR introduce a breaking change?**
No

 

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**
yes

### CODE REVIEW

File: lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart [hunk 1]
```diff
@@ -20,7 +20,8 @@ import 'package:talawa/services/user_config.dart';
 import 'package:talawa/view_model/base_view_model.dart';
 import 'package:talawa/view_model/lang_view_model.dart';
 import 'package:talawa/widgets/custom_alert_dialog.dart';
-import 'package:talawa/widgets/custom_progress_dialog.dart';
+
+import '../../../widgets/custom_progress_dialog.dart';
```
Comment (SiddheshKukade) [inline lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart:24]: Refractor: This is the conventional way of importing packages.

File: lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart [hunk 2]
```diff
@@ -84,9 +85,16 @@ class ProfilePageViewModel extends BaseModel {
...
-                Firebase.app()
-                    .delete(); // Deleting app will stop all Firebase plugins
+                try {
+                  Firebase.app()
+                      .delete(); // Deleting app will stop all Firebase plugins
+                } catch (e) {
+                  debugPrint(
+                      "ERROR: Unable to delete firebase app" + e.toString());
+                }
+
```
Comment (SiddheshKukade) [inline lib/view_model/after_auth_view_models/profile_view_models/profile_page_view_model.dart:95]: Fix: Firebase throws error if it's not initialized previously. Added a `try {..} catch{..} ` to fix it,

**Comment by palisadoes** [comment]
Closing. You are not assigned the issue.

**Comment by palisadoes** [comment]
Reopening as the assignee asked that you work on it.

**Comment by SiddheshKukade** [comment]
@palisadoes  Sir the code is ready to be reviewed. 
Here the codecov patch is failing due to new lines being introduced and not covered in the current test files. I think there can be a separate issue for that. But the current changes are sufficient to fix bug from #1581.
image

**Review by palisadoes — APPROVED** [review]

**Pull request merged**